crates/zkwasm/src/circuits/etable/allocator.rs

```diff
@@ -322,7 +345,15 @@ impl<F: FieldExt> EventTableCellAllocator<F> {
                     bit_table.configure_in_table(
                         meta,
                         "c8f: bit_table_lookup in bit_table",
-                        |meta| curr!(meta, col),
+                        |meta| {
```

Comment:
This implementation assumes that step size >= 4. It is wrong. We should reserve 4 unlimited cell for lookup instead of a column.